host-configs/tioga-toss_4_x86_64_ib_cray-cce@19.0.0_hip.cmake

```diff
@@ -53,6 +53,8 @@ set(MPI_CXX_COMPILER "/usr/tce/packages/cray-mpich-tce/cray-mpich-8.1.32-rocmcc-
 
 set(MPI_Fortran_COMPILER "/usr/tce/packages/cray-mpich-tce/cray-mpich-8.1.32-rocmcc-6.3.1-cce-19.0.0/bin/mpif90" CACHE PATH "")
 
+# Unable to determine MPIEXEC, axom tests may fail
```

Comment:
This is sort of strange since the host-config has an `MPIEXEC_EXECUTABLE` below.

This appears to be the relevant logic in spack: 
https://github.com/spack/spack/blob/4d563acd1b0dd7b21f7df4c1ad756b6f4bdd5b1d/var/spack/repos/spack_repo/builtin/build_systems/cached_cmake.py#L253-L269

Reply:
Looks like we get the `MPIEXEC_EXECUTABLE` from here:

https://github.com/LLNL/axom/blob/dbce3d461849964a6194f4991ccde35d060d8b61/scripts/spack/packages/axom/package.py#L466-L490

Should we add a comment to ignore the previous warning :)
```cmake
            entries.append("# "Trust me -- the other one is lying"\n")
```

Reply:
This logic in Spack has recently changed, Tara, Greg, and I had many conversations how how to do it correctly. We should remove our deletion if you are after this commit in spack: https://github.com/spack/spack/pull/49033

or check if its empty before deleting it: https://github.com/LLNL/serac/pull/1381